### PR TITLE
Lyrics track progress bar color is backwards from GPM UI track progress bar #1800

### LINKED ISSUE #1800: Lyrics track progress bar color is backwards from GPM UI track progress bar
Windows 10 Home, Build 1607

Lyrics overlay seems to cause an issue with the playback scrubber. It appears below the shutter for lyrics, but does not have the timestamp nor media controls. I would assume this is the playback scrubber as it is the only orange element I have on the player that would move to the right in this fashion.
Here is a video of it.
https://www.dropbox.com/s/3hpyedcbevjjp4y/Desktop%2010.19.2016%20-%2018.57.08.01.mp4?dl=0
(sorry for choppy video, Dropbox didn't retain the 50mbps bitrate and 1440p60 resolution)

**Steps to Reproduce:**
Open GPMDP v4.0.1 (latest as of 10/19/16)
Play a song.
Click the 3 dots and select "Show Lyrics (Beta)"
Look at bottom of screen, you can see this:
View

### PR DESCRIPTION
**Pull request by parthsdoshi**
Hardcoded the default color (not sure if the rgb value of default theme is passed through props). Other than that, it seems to work!

Edit: to clarify, this issue fixes #1800

### CODE REVIEW

File: src/renderer/ui/components/generic/LyricsViewer.js [hunk 1]
```diff
@@ -139,15 +139,17 @@ class LyricsViewer extends Component {
 
   render() {
     const barStyle = {};
+    // would rather not hardcode for default theme but not sure if it's passed through any props
+    // dynamically get from webpage?
+    barStyle.backgroundColor = 'rgb(252, 88, 37)';
```
Comment (MarshallOfSound) [inline src/renderer/ui/components/generic/LyricsViewer.js]: There are constants for these values in a LESS file somewhere, not sure if they are in JS anywhere.  Regardless I am happy with them being defined here for now 👍

**Comment by MarshallOfSound** [comment]
> -0.01%

Dayum `coveralls` is harsh, marking a fail for 0.01% 😆

File: src/renderer/ui/components/generic/LyricsViewer.js [hunk 2]
```diff
@@ -139,15 +139,17 @@ class LyricsViewer extends Component {
 
   render() {
     const barStyle = {};
+    // would rather not hardcode for default theme but not sure if it's passed through any props
+    // dynamically get from webpage?
```
Comment (MarshallOfSound) [inline src/renderer/ui/components/generic/LyricsViewer.js]: Can we remove this comment and then I'll be happy to merge this 👍

**Comment by parthsdoshi** [comment]
@MarshallOfSound Does this seem fine? Let me know if you'd like me to add something else!

**Comment by MarshallOfSound** [comment]
LGTM :+1:

**Pull request merged**